src/main/scala/nl.knaw.dans.easy.download/components/AuthenticationComponent.scala

```diff
@@ -69,6 +69,8 @@ trait AuthenticationComponent extends DebugEnhancedLogging {
           .tried
       }.recoverWith {
         case t: AuthenticationException => Failure(InvalidUserPasswordException(userName, new Exception("invalid password", t)))
+        case t: InvalidNameException => Failure(InvalidNameAuthenticationException(t))
+        case t: OperationNotSupportedException if (password.isEmpty) => Failure(NoPasswordAuthenticationException(t))
```

Comment:
Shouldn't this become an internal error? It looks an implementation problem to me.

Reply:
No, this in not internal error.  `Ldap` returns `InvalidNameException` when `user-id is empty`. And it returns `OperationNotSupportedException` when `password is empty`.